dist/images/cleanup.sh

```diff
@@ -6,7 +6,8 @@ kubectl patch svc -n kube-ovn ovn-nb --type='json' -p '[{"op": "replace", "path"
 kubectl patch svc -n kube-ovn ovn-sb --type='json' -p '[{"op": "replace", "path": "/metadata/finalizers", "value": []}]' || true
 
 # Delete Kube-OVN components
-kubectl delete ns kube-ovn
+kubectl delete -f ovn.yaml
```

Comment:
The yaml file may not exist in the current folder. Use url points to the yaml might better